wire-ios-data-model/Support/Sources/MockSafeCoreCrypto.swift

```diff
@@ -20,11 +20,11 @@ import Foundation
 import WireDataModel
 import WireCoreCrypto
 
-class MockSafeCoreCrypto: SafeCoreCryptoProtocol {
+public class MockSafeCoreCrypto: SafeCoreCryptoProtocol {
```

Comment:
suggestion: the file is located in `Sources`, that I think is ok as long as we protect it in any case, that is not used in production. In the past I used precompiled statements, but maybe there is another way?

```
#if DEBUG

public class MockSafeCoreCrypto: SafeCoreCryptoProtocol {
// ...
}

#endif
```

Reply:
Maybe I can move it to `WireDataModelSupport`

Reply:
Ah it's already part of the `WireDataModelSupport` target so it's not going into production

Reply:
Ups ok, didn't see it! 👍